### ISSUE TITLE
Firewall management IP configuration should only be created while creating a new firewall

### ISSUE THREAD

**Issue by iatodoro** [Azure/azure-cli-extensions#1270]
- If the issue is to do with Azure CLI 2.0 in-particular, create an issue here at [Azure/azure-cli](https://github.com/Azure/azure-cli/issues)

### Extension name (the extension in question)
Network (firewall)

### Description of issue (in as much detail as possible)
Currently the only way to attach a Management IP Configuration appears to be with this workflow:
az network firewall create 
az network firewall ip-config create
az network firewall management-ip-config create

Each of these commands executes a PUT against firewall. 
However, this cannot work, as creating a firewall with forced tunneling requires both the “data” and “management” IP configs to be attached to the firewall at the same time.

Attaching the data ip-config first, and then management ip-config later becomes a migration scenario ion the back end, which is not supported today.

Further, we need to prevent customers from attempting to modify already existing firewalls and attaching management IP configs to them, for the same reason.
(There is work planned to block this scenario in NRP, but it’s not there yet.)

Basically, attaching a management IP configuration should only be possible one time, at firewall creation time, and it requires both types of ip-config in one PUT call.

Customers should be able to add more “data” IP configs. (but only a single “management”  IP config)
Additionally, customers should be able to change the public IP referenced in the “management” IP config. 

-----

**Comment by arrownj** [comment]
Hi @MyronFanQiu , could you please help take a look at it?

**Comment by yonzhan** [comment]
add to S165.
